Add dashboard for Kubernetes pods (#66)
appsignal/public_config: dashboards/kubernetes/README.md

> There is currently one automated dashboard for AppSignal for Kubernetes:

Reviewer: ```suggestion
The following automated dashboards exist for AppSignal for Kubernetes:
```
